node/config/def.go

```diff
@@ -144,6 +144,10 @@ type SealingConfig struct {
 	// time buffer for forceful batch submission before sectors/deals in batch would start expiring
 	CommitBatchSlack Duration
 
+	// network BaseFee below which to stop doing commit aggregation, instead
+	// submitting proofs to the chain individually
+	AggregateAboveBaseFee types.FIL
```

Comment:
add the default unit please? attofil?nanofil?.. including a table in the pr description on how to set the value in each unit?

Reply:
It's basefee in FIL (this is what's types.FIL is)